DNN Platform/Modules/ResourceManager/Settings.ascx.cs

```diff
@@ -71,6 +71,7 @@ public override void UpdateSettings()
             }
             catch (Exception exc)
             {
+                // Module failed to load
```

Comment:
Do we really need this comment ? The line right under it is pretty self explanatory...

Reply:
This was left over from the original code, didn't realize you removed it. I'll fix it in the next code push